lib/fluent/plugin/out_influxdb.rb

```diff
@@ -132,6 +132,12 @@ def write(chunk)
         tags[@sequence_tag] = @seq
         @prev_timestamp = timestamp
       end
+
+      if values.empty?
+          $log.warn "Skip record '#{record}', because InfluxDB requires at least one value in raw"
```

Comment:
`$log` is global and `log_level` parameter in plugin doesn't affect it.
So if user set `log_level error` in plugin configuration, it is ignored.
In configuration, this is no problem because configuration warning should be logged in fluentd log.

> who provides log

Plugin itself